### PR TITLE
make test-cluster multi process parallel safe

### PR DESCRIPTION
**Pull request by Raynos**
RTAPI runs multiple processes with multiple concurrent
hyperbahn clusters.

r: @rf @kriskowal @jcorbin

### CODE REVIEW

**Comment by kriskowal** [comment]
:ok:

File: test/lib/remote-config-file.js [hunk 1]
```diff
@@ -33,9 +33,15 @@ function RemoteConfigFile(name) {
...
 
     var self = this;
-    name = name || 'UNKNOWN[' + process.pid + ']';
-    self.filePath = path.join('/tmp',
-                              'remote_config_' + name + '.json');
+
+    if (!name) {
+        name = 'UNKNOWN[' + process.pid + ']';
+    } else {
+        name += '[' + process.pid + ']';
```
Comment (jcorbin) [inline test/lib/remote-config-file.js]: how about the simpler:
```
var name = name || 'UNKNOWN';
name += '[' + process.pid + ']';
```

File: test/lib/remote-config-file.js [hunk 2]
```diff
@@ -33,9 +33,15 @@ function RemoteConfigFile(name) {
...
+    }
+    self.filePath = path.join(
```
Comment (jcorbin) [inline test/lib/remote-config-file.js]: no change on these lines, y u churn

**Comment by jcorbin** [comment]
sure... but could be simpler

**Comment by Raynos** [comment]
Thanks @jcorbin I made it less churny :)

**Pull request merged**